src/util.py

```diff
@@ -263,3 +263,21 @@ def check(reaction, user):
                 pass
             await block_message.edit(content=r"```Closed.```")
             current_index = None
+
+async def err_not_implemented(ctx):
+
+    embed = discord.Embed(
+        title="Sorry!",
+        description=f"This command or feature hasn't been implemented yet! " + \
+            f"Why not DM `queen tired#1745` and harass her about it?" + \
+            f"\n\n" + \
+            f"Make sure to include this text: " + \
+            f"```{ctx.data}```\n\n" + \
+            f"You can use the regular, prefixed version of this command " + \
+            f"until it's added.",
```

Comment:
```suggestion
        description=(f"This command or feature hasn't been implemented yet! "
            f"Why not DM `queen tired#1745` and harass her about it?"
            f"\n\n"
            f"Make sure to include this text: "
            f"```{ctx.data}```\n\n"
            f"You can use the regular, prefixed version of this command "
            f"until it's added.",
        )
```

Reply:
Merged change. For whatever reason, pylint was complaining if I didn't do this.